docs/getting-started/dependencies-for-some-community-packages.md

```diff
@@ -9,34 +9,218 @@ This is an advanced topic and it might not be necessary! Most community packages
+
+Otherwise, here are some things to try:
+
+#### Install `setuptools` for Python
+
+`ppm` may complain about a lack of `distutils`. That’s something we can fix by installing Python’s `setuptools` package.
+
+Run this command in your terminal:
+
+```powershell
+pip install setuptools
+```
+
+If it doesn’t know what `pip` is, then your Python path is not set up properly. Hopefully, it’ll succeed at installing `setuptools`, at which point you should run `ppm install --check` again. If it still fails, keep reading.
+
+#### Tell `ppm` about the path to your version of Python
+
+If `ppm` still complains about missing `distutils` after the step above, or if it thinks you don’t have any build tools installed, it might be using a different version of Python than the one we just installed.
+
+Run `ppm --version` again and look at the version of Python it reports: if it’s not the version you just installed, then we’ll need to tell it how to find your Python installation.
+
+You can tell it about your Python executable two different ways, but let’s first try this one:
+
+```powershell
+ppm config set python "C:\Users\(my-user)\AppData\Local\Programs\Python\Python312\python.exe"
+```
+
+As always, you should make sure this matches the location of your own Python installation.
+
+After you run this command, try `ppm --version` again. If it picks up on your new Python version, you’ve succeeded:
+
+![ppm --version](/img/atom/ppm-version-windows.png)
+
+Otherwise, you can try the other way of telling `ppm` about your Python location: the `PYTHON` environment variable. Run this in your terminal, again confirming you’re using the correct path to your own installation of Python:
+
+```powershell
+$env:Python = "C:\Users\(my-user)\AppData\Local\Programs\Python\Python312\python.exe"
+```
+
+If `ppm` recognizes your Python version when you run `ppm --version`, you should try running `ppm install --check` again. If it passes, you’ve succeeded!
+
+:::tip
+If this second technique succeeded, then you’ll want to make sure that `PYTHON` is defined automatically in future shells as well. You can define it using the same process we documented in the earlier [Adding terminal commands](/getting-started/adding-terminal-commands/#windows) article.
+:::
+
+#### Ensure you installed your Visual Studio tools correctly
+
+If `ppm` _still_ thinks you don’t have the right build tools installed, then it’s worth re-checking whether you’ve installed your Visual Studio tools correctly.
+
+Re-run the Visual Studio installer and confirm:
+
+* You _must_ have **Visual Studio 2022 Build Tools** or **Visual Studio 2019 Build Tools** installed — or the full Visual Studio IDE of either version.
+* Select whichever one you have installed and click on [[Modify]].
+* You _must_ have “Desktop development with C++” checked. If it is not selected, select it, then click on the [[Modify]] button at the bottom right corner of the window. It will install new components.
```

Comment:
```suggestion
* You _must_ have “Desktop development with C++” checked. If it is not selected, select it, then click on the [[Modify]] button at the bottom right corner of the window. It will install new components.
* If your Windows version is something other than Windows 10, ensure you've also checked to install the "Windows 10 SDK"
```

Reply:
You're saying that Windows 11 users will need to install “Windows 10 SDK” just to be able to build a package like `x-terminal-reloaded`?

Reply:
From what I recall, yes.

If memory serves there was a support request of someone attempting to setup their toolchain, and they were on Windows 11. But even after installing everything else it wouldn't work, and they had to install the Windows 10 SDK to proceed.

I'll see if I can double check this info, as I'm just relying on my memory, and the fact that I made sure to add that in the docs after that support interaction. But let me see if I can find a source

Reply:
Alright, @savetheclocktower I've found my original source.

From [this](https://discord.com/channels/992103415163396136/992103415163396139/1128475488092946573) interaction, the user was required to install the "Windows 10 SDK" while they were on Windows 11. This looks to be when attempting to install Spiker's terminal package.

But, and apparently I've forgotten about [this](https://discord.com/channels/992103415163396136/992103415163396139/1134993183953141770). But a while back when we bumped the `node-gyp` version we used, we enabled support of both the Windows 10 SDK and 11. So while my comment was true at one point it no longer is.

So my added docs are now inaccurate. May be a good idea to instead change them to install any SDK version. But if that's not your experience like you said before, it may not be needed and we can omit it entirely. Then if in the future we find some edge cases those can be updated as needed.